modules/ossm-migrating-network-policies-security.adoc

```diff
@@ -0,0 +1,210 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-network-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-network-policies-security_{context}"]
+= Migrating network policies for security reasons
+
+This procedure applies only if you must recreate your network policies first for security reasons.
+
+[IMPORTANT]
+====
+During the recreation of network policies from {SMProduct} 2 to {SMProduct} 3, both control planes must have access to all workloads and vice versa.
+====
+
+.Prerequisites
+
+* You have deployed {ocp-product-title} 4.14 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+
```

Comment:
- You have the OpenShift Service Mesh 2.6.6 Operator installed with the 2.6 SMCP with the manageNetworkPolicy: true
- You have deployed the "bookinfo" and "bookinfo2" apps.